app/user.go

```diff
@@ -2134,11 +2134,11 @@ func (a *App) userBelongsToTeams(userId string, teamIds []string) (bool, *model.
 }
 
 func (a *App) userBelongsToChannels(userId string, channelIds []string) (bool, *model.AppError) {
-	result := <-a.Srv.Store.Channel().UserBelongsToChannels(userId, channelIds)
-	if result.Err != nil {
-		return false, result.Err
+	result, err := a.Srv.Store.Channel().UserBelongsToChannels(userId, channelIds)
```

Comment:
Here you can just `return a.Srv.Store.Channel().UserBelongsToChannels(userId, channelIds)`

Reply:
Done.